docs-new/docs/getting-started/metrics.md

```diff
@@ -77,9 +77,50 @@ and the URL of the target server.
 If the target server is protected by a Secret,
 provide information about the token and key.
 
-For an example of how to specify a KeptnMetricsProvider with a Secret,
-refer to the [example section](../reference/crd-reference/metricsprovider.md#examples)
-of the KeptnMetricsProvider CRD reference page.
+The [keptn-metrics-provider.yaml](../reference/crd-reference/metricsprovider.md#examples)
+file for our example looks like:
+
+```yaml
```

Comment:
it would be nice if you could save the yaml as a separate file asset and import it with something like this

`{% include "./aassets/multiple-metric-providers.yaml" %}`

Reply:
> it would be nice if you could save the yaml as a separate file asset and import it with something like this
> 
> ```yaml
> {% include "./analyzing-application-performance/metric-providers.yaml" %}
> ```
> 
> checkout this example https://raw.githubusercontent.com/keptn/lifecycle-toolkit/main/docs-new/blog/posts/analyzing-application-performance.md

@RealAnna should I also do that for the KeptnMetric example or only the KeptnMetrics Provider?

Reply:
@Vickysomtee in general we are migrating towards this approach so that all yaml can be lited by our pipeline. So please any yaml content should be imported 😺

Reply:
@RealAnna I could use your help in figuring out the proper directory to create the yaml files for this particular PR before importing. I am a bit sceptical.

Reply:
@Vickysomtee usually we keep the asset folder next to where it is used: for instance if you have a mydoc.md file that imports a yaml in https://github.com/Vickysomtee/keptn-lifecycle-toolkit/tree/docs-code-snippet/docs-new/docs/getting-started

you could put the yaml in https://github.com/Vickysomtee/keptn-lifecycle-toolkit/tree/docs-code-snippet/docs-new/docs/getting-started/assets